src/main/resources/db/migration/V1_57__appointment_delivery.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TYPE appointment_delivery_type AS ENUM ('PHONE_CALL','VIDEO_CALL', 'IN_PERSON_MEETING_PROBATION_OFFICE', 'IN_PERSON_MEETING_OTHER');
+
+create table appointment_delivery(
+    appointment_id uuid not null,
+    appointment_delivery_type appointment_delivery_type not null,
+    nps_office_code VARCHAR(7) null,
```

Comment:
can I be annoying and ask to update https://github.com/ministryofjustice/hmpps-interventions-service/blob/main/src/main/resources/db/migration/R__data_dictionary.sql as well, please? Without URNs, it would be important with cross-system IDs to explain where the source of truth is

Reply:
yes I will do that

Reply:
I have now included that in the PR